code/game/objects/items/weapons/clown_items.dm

```diff
@@ -143,3 +143,59 @@
 	attack_verb = list("quacked")
 	hitsound = 'sound/items/quack.ogg'
 	honk_delay = 10
+
+#define GLUE_WEAROFF_TIME -1 //was 9000: 15 minutes, or 900 seconds
+
+/obj/item/weapon/glue
+	name = "bottle of superglue"
+	desc = "A small plastic bottle full of superglue."
+
+	icon = 'icons/obj/items.dmi'
+	icon_state = "glue0"
+
+	w_class = 1
+
+	var/spent = 0
+
+/obj/item/weapon/glue/examine(mob/user)
+	..()
+	if(Adjacent(user))
+		user.show_message("<span class='info'>The label reads:</span><br><span class='notice'>1) Apply glue to the surface of an object<br>2) Apply object to human flesh</span>", MESSAGE_SEE)
+
+/obj/item/weapon/glue/update_icon()
+	..()
+	icon_state = "glue[spent]"
+
+/obj/item/weapon/glue/afterattack(obj/item/target, mob/user, flag)
+	if(!flag)
+		return
+
+	if(spent)
+		user << "<span class='warning'>There's no glue left in the bottle.</span>"
+		return
+
+	if(!istype(target)) //Can only apply to items!
+		user << "<span class='warning'>That would be such a waste of glue.</span>"
+		return
+
+	user << "<span class='info'>You gently apply the whole bottle of [src] to \the [target].</span>"
+	spent = 1
+	update_icon()
+	apply_glue(target)
+
+/obj/item/weapon/glue/proc/apply_glue(obj/item/target)
+	target.cant_drop = 1
+	target.canremove = 0
+
+	if(GLUE_WEAROFF_TIME > 0)
+		spawn(GLUE_WEAROFF_TIME)

```

Comment:
I'm 99% sure that it would, but you would need to do src = null before the spawn. (Also it would still fuck up GC on the target I believe but whatever that rarely would happen)